console/console-init/ui/src/Components/ConnectionDetail/ConnectionDetailHeader.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import {
+  PageSection,
+  PageSectionVariants,
+  Split,
+  SplitItem,
+  GridItem,
+  Grid
+} from "@patternfly/react-core";
+import { LockIcon, LockOpenIcon } from "@patternfly/react-icons";
+import "../AddressSpace/AddressSpaceHeader.css";
+export interface ConnectionHeaderDetailProps {
+  hostname: string;
+  containerId: string;
+  protocol: string;
+}
+export const ConnectionDetailHeader: React.FunctionComponent<
+  ConnectionHeaderDetailProps
+> = ({ hostname, containerId, protocol }) => {
+  const generateIcons = () => {
+    switch (protocol) {
+      case "AMQP":
+        return <LockIcon />;
+      default:
+        return <LockOpenIcon />;
+    }
+  };
+  return (
+    <PageSection variant={PageSectionVariants.light}>
+      <Grid>
+        <GridItem rowSpan={12} className="l_split_m_height">
```

Comment:
We also have the "mobile" layout to support. Is this rendering as expected? This plugin can help testing it https://github.com/storybookjs/storybook/tree/master/addons/viewport

Reply:
nope it is not expected, I need to fix this. As i saw the designs today morning. I will raise a another PR with the correct rendering. Will it work?

Reply:
Ok